README.md

```diff
@@ -20,6 +22,30 @@ jobs:
 
 No further action is required.
 
+## Setup using custom token
+
+This approach uses a custom token that you provide.
+
+**Step 1:** Go to `https://github.com/MY_USERNAME/MY_REPOSITORY/settings/secrets`, click on `Add a new secret`, enter `MY_TOKEN` for `Name`, enter the token into the `Value` field, and then click the green `Add secret` button.
```

Comment:
Maybe we could put `TAGBOT_TOKEN` here (instead of writing `MY_TOKEN`)? I think lots of people will copy-paste the following script, so I think it's helpful to have a reasonable default.

It might also be helpful to say how to get a token, e.g.

>To create a custom token, go to <https://github.com/settings/tokens/new>, name it anything you wish, and select either `repo` or `public_repo`, depending on whether or not you will use tagbot with private repos. Save this somewhere secret, like a password, so you can reuse the same token for each tagbot workflow deployment you make.

Reply:
Sure! Want to add that? I just gave you access to my fork

Reply:
Sure, done!